packages/block-editor/src/components/autocomplete/index.js

```diff
@@ -26,7 +29,13 @@ import { withBlockEditContext } from '../block-edit/context';
  */
 export function withFilteredAutocompleters( Autocomplete ) {
 	return ( props ) => {
-		let { completers = [] } = props;
+		let { completers = [], blockName } = props;
```

Comment:
Each time we arrive to this function with undefined completers, completers will be set to a new array reference and useMemo will execute again. Should we have an empty array constant and set the completers default value to it?